hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3AAnalyticsAcceleratorStreamReading.java

```diff
@@ -22,7 +22,10 @@
 import java.io.File;
 import java.io.IOException;
 import java.io.InputStream;
+import java.lang.reflect.Field;
 
+import org.apache.hadoop.fs.contract.s3a.S3AContract;
```

Comment:
this test needs work, you shouldn't need to depend on anything from the contract package

Reply:
ahh my bad, ITestS3AEncryptionSSEC also does this..